landing_page/views.py

```diff
@@ -111,12 +111,20 @@ def search_movies_json(request):
         search_string = body_data.get("search")
 
         if search_string:
-            movies = Movie.objects.filter(name__istartswith=search_string)
-            sorted_movies = sorted(
-                movies,
-                key=lambda movie: sort_by_closeness(search_string, movie),
-                reverse=True,
-            )[:5]
+            if body_data.get("send_all"):
+                movies = Movie.objects.filter(name__icontains=search_string)
+                sorted_movies = sorted(
```

Comment:
i don't love this being the exact same call for both 'ifs', the only difference being one grabs five entries and the other grabs all...
a comment would also be good here, either header or above the if, to differentiate the purposes here. like "send_all is for onboarding and the other is for the search box suggestions". i had to 'ctrl+f' 'send_all' to see where it was used and then, using intuition, be like "ah the other one must be for the searchbox huh"

Reply:
I thought it would be good to have them both here since they both come from searching for a movie, but yeah it can definitely be condensed